app/src/main/java/com/duckduckgo/app/global/view/DaxDialog.kt

```diff
@@ -194,7 +198,7 @@ class TypewriterDaxDialog : DialogFragment(), DaxDialog {
             primaryButtonText: String,
             secondaryButtonText: String? = "",
             toolbarDimmed: Boolean = true,
-            dismissible: Boolean = true,
+            dismissible: Boolean = false,
```

Comment:
Thanks, this is perfect, but I would also change line `53` in this same file. Just to keep this aligned with default property

Reply:
👍  changed!